src/main/java/org/testng/util/Strings.java

```diff
@@ -9,6 +9,14 @@ private Strings() {
     // Utility class. Defeat instantiation.
   }
 
+  public static String repeat(String text, int count) {
```

Comment:
Fyi, the java 11 implementation http://hg.openjdk.java.net/jdk/jdk/file/fc16b5f193c7/src/java.base/share/classes/java/lang/String.java#l2984

Reply:
Fixed this by adding a TODO to take care of getting rid of this custom implementation once we move to using JDK11